crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -889,5 +889,31 @@ $env.config = {
             mode: emacs
             event: { edit: selectall }
         }
+        # Users frequently request a keybinding to add new lines in code
+        # while using REPL. The only reason those keybindings were never shipped
+        # with the initial reedline set is that those combinations of modifiers
+        # with Enter have somewhat spotty support across different platforms/terminals.
+        # So please feel free to uncomment and try the keybindings that will work for you.
+        # {
+        #     name: insert_newline
+        #     modifier: alt     # In case of problems, check if the control or shift modifier works for you
+        #     keycode: enter
+        #     mode: [emacs vi_normal vi_insert]
+        #     event: { edit: insertnewline }
+        # }
+        {
+            name: insert_newline
+            modifier: none
+            keycode: char_o
+            mode: [vi_normal]
+            event: { edit: insertnewline }
+        }
```

Comment:
This binding like that makes no sense, `o` and `O` are operating on a line level (so insert a line below or above relative to the line and not at the current character cursor like `EditCommand::InsertNewline`) furthermore as you mentioned toggle to insert mode.
On top of that I am not sure if the current normal mode parser that operates on the characters without modifiers would even work with that. So adding this vim feature is probably best done in reedline.
```suggestion
```

Reply:
sure, thanks